tests/phpunit/api/v3/LocBlockTest.php

```diff
@@ -60,16 +60,28 @@ public function testCreateLocBlock() {
       'street_address' => '1234567',
     ));
     $params = array(
-      'address_id' => $address['id'],
-      'phone_id' => $phone['id'],
-      'email_id' => $email['id'],
+      'address' => array(
```

Comment:
If we have to change the test here then perhaps we've broken the test contract? Maybe we need to change the check to verify_one_mandatory(array('address_id', 'address'....)